bot/exts/moderation/infraction/infractions.py

```diff
@@ -27,37 +30,49 @@
     from bot.exts.moderation.watchchannels.bigbrother import BigBrother
 
 
+MAXIMUM_TIMEOUT_DAYS = timedelta(days=28)
+TIMEOUT_CAP_MESSAGE = (
```

Comment:
Suggestion:
This can be further improved by adding more context through mentioning which user was being infracted